desktop/core/base_requirements.txt

```diff
@@ -59,7 +60,7 @@ pytidylib==0.3.2
 pytz==2021.3
 PyJWT==2.4.0
 requests-kerberos==0.12.0
-rsa==4.7
+rsa==4.7.2
```

Comment:
Is there a specific reason for using this specific version? Can't we upgrade it to the latest version, 4.9?

Reply:
The package has dependency on rsa=4.7.2 so I changed to that version only.
https://github.com/GoogleCloudPlatform/gcs-oauth2-boto-plugin/blob/b749ea3424c26145277dc3c3987220891324023f/setup.py#L32

Also since this looks like an important package, I didn't want to bump it up higher due to regression concerns.